doc/release-notes.md

```diff
@@ -105,6 +105,11 @@ Low-level changes
 
 RPC
 ---
+- `bitcoind` can process a limited number of simultaneous RPC requests.
+  Previously, if this limit was exceeded, `bitcoin-cli` displayed the string
```

Comment:
I don't think anyone is using the cli as a stable interface. I was more referring to an http-rpc proxy receiving a different HTTP error code when the limit is exceeded.